src/main/java/tutorspet/logic/util/AttendanceRecordListUtil.java

```diff
@@ -181,6 +217,24 @@ private static AttendanceRecordList replaceAttendanceRecordInList(AttendanceReco
         return new AttendanceRecordList(modifiedAttendanceRecordList);
     }
 
+    /**
+     * Returns an {@code AttendanceRecord} where the {@code Attendance} of the {@code studentToRemove} has been removed.
+     * If the {@code studentToRemove} does not exist in the {@code attendanceRecord}, the {@code attendanceRecord} is
+     * returned unchanged.
```

Comment:
Not sure if this would sound better...
```suggestion
     * If the {@code studentToRemove} does not exist in the {@code attendanceRecord}, the {@code attendanceRecord} 
      * remains unchanged.
```